packages/elements/src/lib/code-helpers.ts

```diff
@@ -54,6 +55,8 @@ export function determineLanguage(fileName: string): ProgrammingLanguage | undef
       return ProgrammingLanguage.gherkin;
     case 'svelte':
       return ProgrammingLanguage.svelte;
+    case 'rs':
```

Comment:
Could you add a case in `test/unit/lib/code-helpers.spec.ts` too? That will actually test the highlighting part of prism working